src/PixelFlags.cc

```diff
@@ -107,7 +107,8 @@ PixelFlagsAlgorithm::PixelFlagsAlgorithm(Control const& ctrl, std::string const&
     // Set all the flags that correspond to mask planes anywhere in the footprint
     _anyKeys["EDGE"] = schema.addField<afw::table::Flag>(
             name + "_flag_edge",
-            "Source is outside usable exposure region (masked EDGE or NO_DATA, or centroid off image)");
+            "Source is outside usable exposure region (masked EDGE or NO_DATA, or centroid off image).");
+    _anyKeys["NO_DATA"] = _anyKeys.at("EDGE");  // Also set edge flag for NO_DATA.
```

Comment:
Do I understand correctly that `_anyKeys[K]`, `_centerKeys[K]`, and `_centerAllKeys[K]` are "the key to the catalog flag that should be set if make K is set" (some comments in the class definition would have been helpful)? Should I be worried about the fact that the [config code](https://github.com/lsst/meas_base/blob/04a9dd8c0aad1c3faaccc75ddf2679f927b2838e/src/PixelFlags.cc#L156-L176) can apparently overwrite these keys?

Reply:
Yes, your understanding of those maps is correct.

Good catch that we've never enforced that the config not change the existing key definitions. That should be fixed, but probably not on this ticket.

I can add some comments to the class header on this ticket: this code goes back to circa 2013 (thought my addition of `_centerAllKeys` is obviously newer).